packages/fx-core/src/component/feature/collaboration.ts

```diff
@@ -121,18 +219,19 @@ export class TeamsCollaboration {
   }
 
   private handleError(error: any, ctx: Context): FxError {
-    if (axios.isAxiosError(error)) {
-      const message = JSON.stringify(error.response!.data);
-      ctx.logProvider.error(message);
-      if (error.response!.status >= 400 && error.response!.status < 500) {
-        return new HttpClientError(componentName, message);
-      } else {
-        return new HttpServerError(componentName, message);
-      }
+    if (error.innerError) {
+      const message = error.innerError.message;
+      const fxError =
+        error.innerError.response.status &&
+        error.innerError.response.status >= 400 &&
+        error.innerError.response.status < 500
+          ? new HttpClientError(componentNameTeams, message)
```

Comment:
Seems the innerError is an AxiosError, if yes, the message here may not be helpful. It just says "request failed with status code xxx". Maybe put the `response.data` is better.